cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/AbstractCluster.java

```diff
@@ -224,6 +224,18 @@ protected String getServiceAccountName() {
         return null;
     }
 
+    /**
+     * @return the cluster name
+     */
+    public String getCluster() {
+        return cluster;
+    }
+
+    public String getPersistentVolumeClaimName(int podId) {
```

Comment:
Why is this in `AbstractCluster` and `kafkaPodName`and `zookeeperPodName` are not here as `getPodName` as well?

Reply:
Yes I can move them.